src/pgduckdb_duckdb.cpp

```diff
@@ -117,6 +117,9 @@ DuckdbCreateConnection(List *rtables, PlannerInfo *planner_info, List *needed_co
 		if (is_r2_cloud_secret) {
 			appendStringInfo(secret_key, ", ACCOUNT_ID '%s'", secret.endpoint.c_str());
 		}
+		if (!secret.use_ssl) {
+			appendStringInfo(secret_key, ", USE_SSL '%s'", "FALSE");
```

Comment:
No reason to format here:

```suggestion
			appendStringInfo(secret_key, ", USE_SSL 'FALSE'");
```